sycl/include/CL/sycl/detail/spirv.hpp

```diff
@@ -153,9 +153,9 @@ EnableIfGenericBroadcast<T, IdT> GroupBroadcast(T x, IdT local_id) {
   char *ResultBytes = reinterpret_cast<char *>(&Result);
   auto BroadcastBytes = [=](size_t Offset, size_t Size) {
     uint64_t BroadcastX, BroadcastResult;
-    detail::memcpy(&BroadcastX, XBytes + Offset, Size);
+    std::memcpy(&BroadcastX, XBytes + Offset, Size);
```

Comment:
It look like this breaks the tests due to missing `#include <cstring>`. See pre-commit failures for https://github.com/intel/llvm/pull/4153.
@Pennycook, could you fix this ASAP, please?

Reply:
Sorry about that.  Opened https://github.com/intel/llvm/pull/4157 to (hopefully) fix it.